package.json

```diff
@@ -1,30 +1,54 @@
 {
   "name": "probot",
-  "version": "0.0.0-development",
-  "description": "a trainable robot that responds to activity on GitHub",
+  "version": "6.1.0-typescript",
+  "publishConfig": {
+    "tag": "next"
+  },
+  "description": "a trainable robot that responds to activity on GitHub.",
   "repository": "https://github.com/probot/probot",
   "main": "lib/index.js",
   "bin": {
     "probot": "./bin/probot.js"
   },
   "scripts": {
+    "build": "rimraf lib && tsc -p tsconfig.json",
+    "start": "node ./bin/probot run",
     "test": "jest --coverage && standard && npm run doc-lint",
     "doc-lint": "standard docs/**/*.md",
     "doc": "jsdoc -c .jsdoc.json",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
-    "postsemantic-release": "script/publish-docs"
+    "postsemantic-release": "script/publish-docs",
+    "prepublish": "npm run build"
   },
   "jest": {
```

Comment:
I would consider moving this to a `jest.config.js` file in the root of the project.  package.json is already quite long.